apis/core/v1alpha1/flagsourceconfiguration_types.go

```diff
@@ -128,6 +128,10 @@ type FlagSourceConfigurationSpec struct {
 	// DebugLogging defines whether to enable --debug flag of flagd sidecar. Default false (disabled).
 	// +optional
 	DebugLogging *bool `json:"debugLogging"`
+
+	// Resources defines flagd sidecar resources. Default to operator sidecar-cpu-limit and sidecar-ram-limit flags.
+	// +optional
+	Resources corev1.ResourceRequirements `json:"resources"`
```

Comment:
Similar to what you did in PR https://github.com/open-feature/open-feature-operator/pull/513, you can add these to v2 & v3 APIs.